JamesKirkAndSpock/Hirola
# Materialize test

## #156062740 Make mockups for the website pages
### Purpose of this PR
In this PR  I have created the landing page in materialize css, Its responsive in most screens 
#### Description of Task to be completed?
The index page of the site is functional
#### How should this be manually tested?
Check out the hosted app [here](http://test.crispuskamau.com/?)
Use chrome to view the dispay on different screen sizes
#### What are the relevant pivotal tracker stories?
* #156062740
#### Screenshots (if appropriate)
#### Questions:
